docs/concepts.rst

```diff
@@ -240,6 +240,19 @@ We can put this all together to build a simple pipeline:
                 python_callable=lambda: print("GOODBYE!"))
         )
 
+Bitshift can also be used with lists. For example:
+
+.. code:: python
+
+    op1 >> [op2, op3]
+
+is equivalent to:
+
+.. code:: python
+
+    op1.set_downstream(op2)
```

Comment:
More correct, what this is equivalent to op1.set_downstream([op2, op3])

Reply:
@xnuinside There is no difference. 

All these 3 are equivalent:

1. 
 `op1.set_downstream([op2,op3])`

2. 
```
op1.set_downstream(op2)
op1.set_downstream(op3)
```

3. 
`op1 >> [op2, op3]`


![first](https://user-images.githubusercontent.com/40163787/48539385-9f12b100-e8bf-11e8-88af-91055cda7def.PNG)

The idea was to explain that lists are supported with bit-shift (3). I wasn't trying to show all possible ways to set dependencies. Explaining about (1) vs (2) might be better in setting up dependencies section
https://airflow.apache.org/tutorial.html#setting-up-dependencies

Reply:
There is difference: the way what you use you create illusion what there are no possible to upstream/downstream a list of task with methods and it’s more simple to use bitwise list when use function for each task - this is what you show on such example

Reply:
@ron819, why you don't want to add info about this in doc also? I mean about 'set_downstream' :) I believe everyone will be grateful to you

Reply:
@xnuinside I asked if you think it's better to add it in https://airflow.apache.org/tutorial.html#setting-up-dependencies however you did not comment on this :)  I can surly add this just wondering where is the best place for it.

Reply:
@ron819 oh, sorry, I miss this :) yes, better to add this, and thank you! https://airflow.apache.org/tutorial.html#setting-up-dependencies - looks good for that, under existing examples with a comment what, possible use with a list of tasks

Reply:
@xnuinside well looking at the updated master docs:
https://airflow.readthedocs.io/en/latest/tutorial.html#setting-up-dependencies
it's already there :)

Reply:
@ron819 it's crazy.. )) why we have two inconsistent docs without versions?! ))

Reply:
@xnuinside I know that @kaxil started a discussion on mailing list to resolve the 2 docs websites issue. any way not part of this tiny PR :)

I don't know why tests are failing for the 2nd commit but it's unrelated to this PR.

So if no other comments this can be merged :)